microprofile-health/src/test/java/org/jboss/eap/qe/microprofile/health/integration/FailSafeCDIHealthBaseTest.java

```diff
@@ -0,0 +1,350 @@
+package org.jboss.eap.qe.microprofile.health.integration;
+
+import static io.restassured.RestAssured.get;
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.containsInAnyOrder;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.hasItems;
+import static org.hamcrest.Matchers.hasSize;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.not;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.TimeoutException;
+
+import org.jboss.arquillian.container.test.api.RunAsClient;
+import org.jboss.eap.qe.microprofile.health.tools.HealthUrlProvider;
+import org.jboss.eap.qe.microprofile.tooling.server.configuration.ConfigurationException;
+import org.jboss.eap.qe.microprofile.tooling.server.configuration.arquillian.ArquillianContainerProperties;
+import org.jboss.eap.qe.microprofile.tooling.server.configuration.arquillian.ArquillianDescriptorWrapper;
+import org.jboss.eap.qe.microprofile.tooling.server.configuration.creaper.ManagementClientProvider;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.wildfly.extras.creaper.core.online.OnlineManagementClient;
+import org.wildfly.extras.creaper.core.online.operations.admin.Administration;
+
+import io.restassured.http.ContentType;
+import io.restassured.response.ValidatableResponse;
+import io.restassured.specification.RequestSpecification;
+
+public abstract class FailSafeCDIHealthBaseTest {
+    protected RequestSpecification metricsRequest;
+
+    @Before
+    public void before() throws ConfigurationException, InterruptedException, TimeoutException, IOException {
+        ArquillianContainerProperties arqProps = new ArquillianContainerProperties(
+                ArquillianDescriptorWrapper.getArquillianDescriptor());
+        String url = "http://" + arqProps.getDefaultManagementAddress() + ":" + arqProps.getDefaultManagementPort()
+                + "/metrics";
+        metricsRequest = given().baseUri(url).accept(ContentType.JSON);
+
+        try (OnlineManagementClient client = ManagementClientProvider.onlineStandalone()) {
+            new Administration(client).reload();
```

Comment:
I guess this is here to have clean state,right? Or can it be omitted?

Reply:
yeah, no needed